api/app/signals/apps/email_integrations/integrations/core.py

```diff
@@ -140,6 +127,37 @@ def create_status_change_notification_split(signal, status):
         'status': status,
     }
 
-    template = loader.get_template('email/signal_status_changed_split.txt')
-    message = template.render(context)
-    return message
+    template = loader.get_template('email/signal_split.txt')
+    txt_message = template.render(context)
+
+    template = loader.get_template('email/signal_split.html')
+    html_message = template.render(context)
+
+    return txt_message, html_message
+
+
+def send_mail_reporter_status_changed_in_behandeling(signal, status):
+    signal_in_behandeling = status.state == workflow.ON_HOLD
```

Comment:
Naming in this function is confusing. 

(ON_HOLD is not IN_BEHANDELING)

Reply:
Fixed